compiler/rustc_mir_dataflow/src/elaborate_drops.rs

```diff
@@ -411,9 +411,10 @@ where
     fn open_drop_for_box(&mut self, adt: ty::AdtDef<'tcx>, substs: SubstsRef<'tcx>) -> BasicBlock {
         // drop glue is sent straight to codegen
         // box cannot be directly dereferenced
-        let unique_ty = adt.non_enum_variant().fields[0].ty(self.tcx(), substs);
-        let nonnull_ty =
-            unique_ty.ty_adt_def().unwrap().non_enum_variant().fields[0].ty(self.tcx(), substs);
+        let unique_ty = adt.non_enum_variant().fields[FieldIdx::new(0)].ty(self.tcx(), substs);
+        let nonnull_ty = unique_ty.ty_adt_def().unwrap().non_enum_variant().fields
+            [FieldIdx::new(0)]
```

Comment:
```suggestion
        let unique_ty = adt.non_enum_variant().fields[FieldIdx::from_u32(0)].ty(self.tcx(), substs);
        let nonnull_ty = unique_ty.ty_adt_def().unwrap().non_enum_variant().fields
            [FieldIdx::from_u32(0)]
```

Reply:
Also `[]` on the new line looks a bit weird, is this rustfmt's formatting? 🤔

Reply:
Yup, this is rustfmt's choice 🤷